frontend/docs/css-style.guide.md

```diff
@@ -0,0 +1,277 @@
+CSS Style guide
+===============
+
+## Purpose
+The aim of this document is to show good practices and conventions while using CSS in Wolox S.A.. This document will be used in trainings and as a reference in Code Reviewing.
+This document must be reviewed and regularly updated.
+
+
+## Style
+#### Naming:
+When naming a class, use names as short as possilbe, but large enough to be representative:
+
+**.navigation** is too large, when we consider that **.nav** is representative enough
+
+**.atr** is very short and not representative enough to name **.author**
+
+Using ids in css (i.e.: _#navigation_) is not recommended.
+
+There are particular ocassions where using ids is the only choice (i.e.: You want to override some rules from a component).
+
+If a classname should have 2 words, they must be separed only by a middle dash (hyphen-case a.k.a. kebab-case)
+
+**.demo-image** instead of **.demo_image**, nor **.demoImage** nor **.demoimage**
+
+#### CSS and HTML tags:
+Avoid using HTML tags en css:
+Instead of **p.error{ }** , use **.error{ }**
+Instead of **p {}**, include in your html **p.caption** and use **.caption{}**
+
+#### Shorthands:
+Shorthands use is expected
+
+    margin: 0 10px 20px;
+
+Shorthands (used for margin, padding, border, etc.) depends on the amount of values listed.
+
+If a shorthand has 4 values
+
+    margin: 25px 50px 75px 100px;
+
+That means
+
+    margin-top: 25px;
+    margin-right: 50px;
+    margin-bottom: 75px;
+    margin-left: 100px;
+
+If a shorthand has 3 values
+
+    margin: 25px 50px 75px;
+
+That means
+
+    margin-top: 25px;
+    margin-right: 50px;
+    margin-bottom: 75px;
+    margin-left: 50px;
+
+If a shorthand has 2 values
+
+    margin: 25px 50px;
+
+That means
+
+    margin-top: 25px;
+    margin-right: 50px;
+    margin-bottom: 25px;
+    margin-left: 50px;
+
+If a shorthand has 1 value
+
+    margin: 25px;
+
+That means
+
+    margin-top: 25px;
+    margin-right: 25px;
+    margin-bottom: 25px;
+    margin-left: 25px;
+
+
+#### Units:
+Avoid adding unit suffix when the value is **“0”**.
+
+    margin: 0;
+
+Instead of
+
+    margin: 0px;
+
+#### Fonts:
+Web safe fonts (A.K.A. callback fonts) are recommended
+
+    .caption {
+        font-family: 'Times New Roman', Times, serif;
+    }
+
+If the browser couldn't load the first font, it would use the next one and so on.
+
+#### Colors:
+Color variables must be used
+
+    color: #4285F4; //wrong
```

Comment:
Just a comment, I think that linters ask you to have a white space to begin the comments, so it would be good that you do it as well in your examples :wink: (`// Wrong`)